### DISCUSSION TITLE
Usage of `csv_column_list`, `csv_indicator`, and `csv_whitelist` tables

### DISCUSSION THREAD

**Discussion by msk** [aicers/review-database#341] [Q&A] [answered]
We've been reviewing the current codebase for the `review-database` crate and noticed that we maintain three tables in the underlying PostgreSQL database: `csv_column_list`, `csv_indicator`, and `csv_whitelist`. From what I can see, these tables were originally added by @sehkone. However, I cannot find any code in the current repository that inserts data into these tables.

Before proceeding with any changes, I wanted to ask if these tables are still being used in any capacity outside of the `review-database` crate. Specifically, are these tables interacted with by any external tools or services that directly interact with PostgreSQL (e.g., custom scripts, third-party tools, etc.)? If these tables are no longer in use, I am planning to remove them in the next version of `review-database`.

Please let me know if there are any ongoing use cases for these tables.

**Comment by sehkone** [comment]
I will check them and let you know if they are required.

**Comment by sehkone** [comment, marked as answer]
@msk 
As you pointed out and expected, they are no longer needed outside the review-database either. Thank you for thoroughly looking into it.
